pkg/R/RDD.R

```diff
@@ -98,48 +138,44 @@ setMethod("getJRDD", signature(rdd = "PipelinedRDD"),
             computeFunc <- function(split, part) {
               rdd@func(split, part)
             }
-            serializedFunc <- serialize("computeFunc", connection = NULL,
+            serializedFuncArr <- serialize("computeFunc", connection = NULL,
                                         ascii = TRUE)
```

Comment:
The `ascii = TRUE` is not introduced in this PR and seems to go a long way back (introduced in https://github.com/amplab-extras/SparkR-pkg/commit/25a0bea75923abb1734afd3f947e6feb5df6d1d3)

I don't recall why we added it back then but I've opened a JIRA to investigate if things work fine without it
https://sparkr.atlassian.net/browse/SPARKR-179